### ISSUE TITLE
How to achieve elementwise convolution for two tensors?

### ISSUE THREAD

**Issue by bugzhu** [tensorflow/tensorflow#34572]
In my problem, I want to convolve two tensors in my neural network model. 

The shape of two tensors is **[None, 2, 1], [None, 3, 1]** respectively. The axis with dimension None means the batch size of the input tensor. For each sample in batch, I want to convolve the two tensors with shape [2, 1] and [3, 1].

However, the tf.nn.conv1d in TensorFlow can only convolve the input with **a fixed kernel.** Is there any function that can support the convolution of two tensors according to the batch size axis, similar to the tf.multiply which can multiply two tensors for each sample or just elementwise multiplication.

**Comment by oanush** [comment]
@bugzhu ,
Hi, can you please provide a standalone code used?Thanks!

**Comment by bugzhu** [comment]
The code I ran can be simplified as follows:
```
input_signal = Input(shape=(L, M), name='input_signal')
input_h = Input(shape=(N), name='input_h')
faded= Lambda(lambda x: tf.nn.conv1d(input, x))(input_h) 
```
What I want to do is that the sample of input_signal can be convolved by the sample of input_h with the same index. However, it just shows my pure idea which can not be able to run in the env. My question is that how I can modify the code to enable the input tensor can be convolved with another input tensor for every sample in the batch.

**Comment by jvishnuvardhan** [comment]
This is not Build/Installation or Bug/Performance issue. Please post this kind of support questions at [Stackoverflow](https://stackoverflow.com/questions/tagged/tensorflow). There is a big community to support and learn from your questions. GitHub is mainly for addressing bugs in installation and performance. Thanks!
